src/fileConvert/fileConvert.ts

```diff
@@ -30,7 +32,11 @@ export const configureFile = async (filename: string) => {
   const archiveName = archiveNamePrefix + formattedDate + '.tar.gz';
 
   try {
-    const data = fs.readFileSync(filename).toString().split('\n');
+    const data = bufferInput.toString().split('\n');
+
+    if (data.length == 1 && data[0] === '') {
+      throw new Error('No data provided');
+    }
 
     const numberOfRecords = data.length.toString();
```

Comment:
You do not need to do `toString()` here.

Also, if you place `numberOfRecords` just after line 35, you can then use it in lines 37, 45 & 48.